Lib/idlelib/editor.py

```diff
@@ -2,7 +2,9 @@
 import importlib.util
 import os
 import platform
+import re
 import string
+import sys
```

Comment:
I am adding these imports to editor.py separately, and backport to 3.8,3.7.  I will then remove them here.  This issue should be backported to 3.8 but not 3.7.

Reply:
OK, do I need to do anything?

Reply:
No, separate PR merged and backports in progress.